torch/_inductor/codegen/rocm/benchmark.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import sys
+
+import pandas
+
+import torch
+from torch._dynamo import config as dynconfig
+from torch._inductor import config
+from torch._inductor.codegen.rocm.ck_template import CKTemplate
+
+log = logging.getLogger(__name__)
+
+
+def generate_inputs(M, N, K, tensor_options, layout, f=torch.randn):
+    if layout[0] == "r":
+        a = f(M, K, **tensor_options)
+    elif layout[0] == "c":
+        a = f(K, M, **tensor_options).transpose(0, 1)
+    else:
+        a = None
+
+    if layout[1] == "r":
+        b = f(K, N, **tensor_options)
+    elif layout[1] == "c":
+        b = f(N, K, **tensor_options).transpose(0, 1)
+    else:
+        b = None
+
+    if layout[2] == "r":
+        out = torch.empty(M, N, **tensor_options)
```

Comment:
I know nvidia hardware is sensitive to the number of 0s in input for mm benchmarking. do you want to do empty here instead of rand ?

Reply:
This is output, should be fine.